app/javascript/controllers/time_zone_controller.js

```diff
@@ -0,0 +1,48 @@
+import { Controller } from "@hotwired/stimulus"
+import lookupFriendlyTimeZone from "util/lookupFriendlyTimeZone"
+
+export default class extends Controller {
+  connect() {
+    const resolvedZone = Intl.DateTimeFormat().resolvedOptions().timeZone
+    const friendlyZone = lookupFriendlyTimeZone(resolvedZone)
```

Comment:
oh, ok. i read ahead to the `lookupFrinedlyTimeZone` util, which is a mapping of the local time zones to it's "general use"(?) counterpart. so maybe that answers my above question?

Reply:
You got it.  And right below this, we change the timezone picker to be either this friendlyZone (`Mountain Time`) or whatever string the browser returned (`America/Denver` style).

And if we have no `resolvedZone` - we leave the field alone.  And UTC will be selected, since that's what the server rendered in the view.